amazon_cloud_code_generator/cmd/refresh_modules.py

```diff
@@ -240,8 +239,9 @@ def main():
     module_list = []
 
     for type_name in RESOURCES:
-        print(f"Generating modules {type_name}")
-        schema_file = args.schema_dir / f"{type_name}.json"
+        file_name = re.sub("::", "_", type_name)
```

Comment:
Not a big deal, but you can also use `str.replace()` https://docs.python.org/3/library/stdtypes.html?highlight=replace#str.replace